critical_demand.py

```diff
@@ -496,7 +494,7 @@ def run_simulation(df_costs, data, settings):
     asset_results = asset_results[RESULTS_COLUMN_NAMES]
     asset_results.to_csv(f"results_{case}.csv")
 
-    NPV = (asset_results.annual_costs.sum() + asset_results.cash_flow.sum()) / CRF
+    NPV = ((asset_results.annual_costs.sum() + asset_results.cash_flow.sum()) / CRF) + project_planning_cost
```

Comment:
it is possible to define it in the input file, so that the user does not need to change this value in the .py code

Reply:
Yes, we could add it later to the input file and let the user define it